app/web/src/features/auth/signup/BasicForm.tsx

```diff
@@ -82,6 +87,11 @@ export default function BasicForm() {
                 value: nameValidationPattern,
               },
               required: NAME_REQUIRED,
+              validate: {
+                startsWithWhiteSpace: (value) => {
+                  return !value.startsWith(" ") || NAME_NOT_TRIMMED;
+                },
+              },
```

Comment:
If we trim the name (which I think is fine, especially since it is easily edited later), I don't think this is needed.

Reply:
I thought it is better to be sure that no one gives white spaces in purpose (which would be odd, but still), but ok, can change